src/main/resources/application.properties

```diff
@@ -51,11 +43,21 @@ logging.level.ca.uhn.fhir.rest.client.impl=INFO
 #fhir-server-configuration.redirect=http://localhost:8080/hpo_on_fhir/authorize
 #fhir-server-configuration.client-id=6c5790e4-bc54-47a9-8c74-0a7b42e0ef91
 
-fhir-server-configuration.name=Epic App Orchard
-fhir-server-configuration.url=https://apporchard.epic.com/interconnect-aocurprd-oauth/api/FHIR/DSTU2
-fhir-server-configuration.version=STU2
-fhir-server-configuration.authorize=https://apporchard.epic.com/interconnect-aocurprd-oauth/oauth2/authorize
-fhir-server-configuration.token=https://apporchard.epic.com/interconnect-aocurprd-oauth/oauth2/token
-fhir-server-configuration.redirect=https://octridev.ohsu.edu/hpo_on_fhir/authorize
-fhir-server-configuration.client-id=7a27ae5f-897b-4997-bd8a-de7fcbdd04a3
-fhir-server-configuration.client-secret=${CLIENT_SECRET:secret}
+#fhir-server-configuration.name=Epic App Orchard
+#fhir-server-configuration.url=https://apporchard.epic.com/interconnect-aocurprd-oauth/api/FHIR/DSTU2
+#fhir-server-configuration.version=STU2
+#fhir-server-configuration.authorize=https://apporchard.epic.com/interconnect-aocurprd-oauth/oauth2/authorize
+#fhir-server-configuration.token=https://apporchard.epic.com/interconnect-aocurprd-oauth/oauth2/token
+#fhir-server-configuration.redirect=https://octridev.ohsu.edu/hpo_on_fhir/authorize
+#fhir-server-configuration.client-id=${FHIR_SERVER_CLIENT_ID}
+#fhir-server-configuration.client-secret=${FHIR_SERVER_CLIENT_SECRET:none}
+
+fhir-server-configuration.name=${FHIR_SERVER_NAME}
+fhir-server-configuration.url=${FHIR_SERVER_URL}
+fhir-server-configuration.version=${FHIR_SERVER_VERSION}
+fhir-server-configuration.authorize=${FHIR_SERVER_AUTHORIZE}
+fhir-server-configuration.token=${FHIR_SERVER_TOKEN}
+fhir-server-configuration.redirect=${FHIR_SERVER_REDIRECT:https://octridev.ohsu.edu/hpo_on_fhir/authorize}
```

Comment:
Yes, we should definitely standardize this in our projects. I am always struggling to get this right and figure out how to override in our various environments (docker vs running as a jar). I'm going to leave as is for now because Dena is actively testing the dev environment and I want it to be as stable as possible. This will never go to prod, so good enough for now.